tests/shared/src/test/scala/scala/meta/tests/transversers/TransformerSuite.scala

```diff
@@ -0,0 +1,56 @@
+package scala.meta.tests.transversers
```

Comment:
no changes to the code below... I just separated the transformer's tests from the traverser's